File: backend/src/main/java/bpr/fitness/companion/backend/model/dto/Exercise.java
package bpr.fitness.companion.backend.model.dto;


import lombok.Data;
import java.util.List;
import java.util.Set;

@Data
public class Exercise {
    private Long id;
    private String name;
    private String description;
    private String category;
    private String imageUrl;
    private Long dayRoutineId;

    private Set<Long> exerciseRecords;
    private List<Long> exerciseDayRoutine;
}

File: FrontEnd/app/src/main/java/com/example/fitnesscompanion_front/model/Exercise.kt
package com.example.fitnesscompanion_front.model

data class Exercise(
    val id:  Int = 0,
    val name: String,
    val category: String,
    val description: String
)


data class ExerciseRecord(
    val exerciseId: Int,
    val repetitions: Int,
    val weight: Float
)
